### DISCUSSION TITLE
[Feature] Shortcut to go to the parent bullet of the current bullet

### DISCUSSION THREAD

**Discussion by misaxi** [vslinko/obsidian-outliner#140] [Ideas]
**Describe the feature**
Would like to have a shortcut (ideally configurable shortcut), to quickly move the cursor to the parent of the current bullet. This will save lots of the keyboard stokes for navigating the structure.

**Comment by vslinko** [comment]
@misaxi Thank you for the idea!

Moved it closer to other ideas.
